### ISSUE TITLE
Baseline 3

### ISSUE THREAD

**Issue by Ad7amstein** [Ad7amstein/deep-activity-recognition#11]
In this baseline, fine-tune `resnet` model to classify each person crop over 9 actions, after fine-tuning `last layer` features are pooled over all people, and are fed to a `softmax` classifier to recognize group activities in each single frame.
